### PR TITLE
Simple styling - Register Form/Sign In Form

### PR DESCRIPTION
**Pull request by EllieChen-Git**
Hi Nina,
I reverted my code back and now my register & sign in functions are working fine (yayyy), and I just quickly styled them with React-Bootstrap.
Btw, I also did some very simple front-end validation about these two forms (People need to enter all the fields in order to submit. I'll try to do the backend validation with celebrate tmr or over the weekend).

I was about to style my image uploading pages as well, but then I realised my 'create apartment form' (image-uploading) is no longer working lol. I think I'll stop here & hope we can figure this out tmr :)
Thanks!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ class App extends Component {
   render() {
     const { user, token } = this.props;
     if (!user && token !== null) {
-      LocalApi.get("/users/me")
+      LocalApi.get("/users/me") //Bug here: need to fix in the future
```
Comment (EllieChen-Git) [inline src/App.js:32]: There's bug about this on the console. I'm not sure what you wanna achieve here, so I don't know if I can fix this. Let's talk about this tmr & see if we can figure it out :)

File: src/components/forms/RegisterForm.js [hunk 2]
```diff
@@ -23,7 +21,6 @@ class RegisterForm extends Component {
     LocalApi.post("users", { username, email, password })
       .then(response => {
         setAuthToken(response.data.token);
-        //      useHistory().push("/");
```
Comment (EllieChen-Git) [inline src/components/forms/RegisterForm.js:26]: Deleted the unnecessary code & comment here (as we discussed on the previous pull request)

File: src/components/forms/RegisterForm.js [hunk 3]
```diff
@@ -40,36 +37,53 @@ class RegisterForm extends Component {
...
-      <div>
-        <form onClick={this.handleClick} onSubmit={this.onFormSubmit}>
-          <label htmlFor="username">Username:</label>
-          <input
-            type="text"
-            value={username}
-            autoComplete="username"
-            onChange={e => this.onInputChange("username", e)}
-          />
+      <Container>
```
Comment (EllieChen-Git) [inline src/components/forms/RegisterForm.js:40]: Register function remain the same. I just styled the form here!

File: src/components/forms/SignInForm.js [hunk 4]
```diff
@@ -32,32 +33,42 @@ class SigninForm extends Component {
...
-            type="password"
-            id="current-password"
-            required="required"
-            autoComplete="current-password"
-            value={password}
-            onChange={e => this.onInputChange("password", e)}
-          />
-          <input type="submit" value="Sign In" />
-        </form>
+        <Form onClick={this.handleClick} onSubmit={this.onFormSubmit}>
```
Comment (EllieChen-Git) [inline src/components/forms/SignInForm.js:38]: Sign in function remain the same. I just styled the form here!

File: src/components/pages/RegisterPage.js [hunk 5]
```diff
@@ -7,7 +7,6 @@ class RegisterPage extends Component {
 
     return (
       <div>
-        <h1>Register a new user</h1>
```
Comment (EllieChen-Git) [inline src/components/pages/RegisterPage.js:10]: As our register page and sign in page are quick simple, I'm thinking that we probably don't need to have 'Register Page' and 'Register Form' separately? Same as 'Sign In Page' (currently empty) and 'Sign in Form'? We can discuss this later as it's not important :)

File: src/App.js (hunk 1 above)
Comment (INRokh) [reply src/App.js:32]: I'll have a look, thanks

File: src/components/forms/RegisterForm.js (hunk 2 above)
Comment (INRokh) [reply src/components/forms/RegisterForm.js:26]: ok, thanks

File: src/components/forms/RegisterForm.js (hunk 3 above)
Comment (INRokh) [reply src/components/forms/RegisterForm.js:40]: cool

**Review by INRokh — APPROVED** [review]

**Pull request merged**